### PR TITLE
RemoteAutocomplete component

### PR DESCRIPTION
**Pull request by Exmirai**
Added RemoteAutocomplete component. Fixed frontmatter validation. Fixed RestoredDataModal text. Updated swagger model. Fixed console error messages

### CODE REVIEW

**Comment by peterliapin** [comment]
@Exmirai please refactor categories to category.

File: src/lib/automapper/index.ts [hunk 1]
```diff
@@ -67,19 +67,15 @@ PojosMetadataMap.create<ContentCreateDto>("ContentCreateDto", {
   type: String,
   author: String,
   language: String,
-  categories: String,
-  tags: String,
+  categories: [String],
```
Comment (peterliapin) [inline src/lib/automapper/index.ts:70]: @Exmirai let's make it category instead of multiple categories.

**Review by peterliapin — CHANGES_REQUESTED** [review]

**Comment by peterliapin** [comment]
@Exmirai please fix merge conflicts.

**Review by gsplakmal — COMMENTED** [review]
Shouldn't the component/feature naming follow the existing standard such as app-header, error-boundary, module-wrapper, etc... ? 
So FileDropdown may be renamed as file-dropdown and likewise.

**Comment by Exmirai** [comment]
There's no global, community accepted naming convention afaik

**Pull request merged**